packages/front-end/src/app/components/EGLabView.vue

```diff
@@ -61,6 +63,7 @@
   type GenericRun = {
     type: 'seqera' | 'omics';
     id: string;
+    runId: string;
```

Comment:
How is `runId` different from `id`? Can this be clarified?

Reply:
`runId` is redundant so I'll remove.